docs/installation_guide.md

```diff
@@ -16,9 +16,25 @@ In addition, the deployment instructions on Kubernetes has been created by the c
 
 ## Prerequisites for the target host
 Harbor is deployed as several Docker containers, and, therefore, can be deployed on any Linux distribution that supports Docker. The target host requires Python, Docker, and Docker Compose to be installed.  
-* Python should be version 2.7 or higher.  Note that you may have to install Python on Linux distributions (Gentoo, Arch) that do not come with a Python interpreter installed by default  
-* Docker engine should be version 1.10 or higher.  For installation instructions, please refer to: https://docs.docker.com/engine/installation/
-* Docker Compose needs to be version 1.6.0 or higher.  For installation instructions, please refer to: https://docs.docker.com/compose/install/
+### Hardware
+|Resource|Capacity|Description|
+|---|---|---|
+|CPU|minimal 2vCPU|4 vCPU is prefered|
```

Comment:
vCPU is for virtual machine, how about physical machine? maybe 2 CPUs is good enough?

Reply:
Fixed, Please help recheck.